doc/sources/patching/patching-options.rst

```diff
@@ -28,6 +28,10 @@
     from sklearnex import patch_sklearn
     patch_sklearn()
 
+.. important::
+
+    Scikit learn import must be done **after** these lines, otherwise the patching will not affect the original scikit-learn estimators.
```

Comment:
```suggestion
    You have to import scikit-learn **after** these lines. Otherwise, the patching will not affect the original scikit-learn estimators.
```